src/core/annotation.js

```diff
@@ -1359,26 +1369,41 @@ class WidgetAnnotation extends Annotation {
     return [initialState.font, initialState.fontName, initialState.fontSize];
   }
 
-  _computeFontSize(font, fontName, fontSize, height) {
+  _computeFontSize(font, fontName, fontSize, height, lineCount) {
     if (fontSize === null || fontSize === 0) {
-      const em = font.charsToGlyphs("M")[0].width / 1000;
-      // According to https://en.wikipedia.org/wiki/Em_(typography)
-      // an average cap height should be 70% of 1em
-      const capHeight = 0.7 * em;
-      // 1.5 * capHeight * fontSize seems to be a good value for lineHeight
-      fontSize = Math.max(1, Math.floor(height / (1.5 * capHeight)));
+      if (lineCount === -1) {
+        fontSize = Math.round(0.8 * height * 10) / 10;
+      } else {
+        // Hard to guess how many lines there are
+        // let's say that fonsize is 10.
```

Comment:
Is this not defined anywhere in the PDF data? This is an assumption that surely doesn't always apply.

Reply:
Sure but need to start from something...
As said I've no idea on the number of lines we've for this textarea and we can't rely on the number we've from the user input.
Any idea is welcome.
I thought that maybe we could try to infer it using the font size used in the page itself, if all the text is 13 for example then it can make sense to have 13 for the input too. But if the page contains different fonts at different sizes....